frappe/hooks.py

```diff
@@ -30,6 +30,7 @@
 	"controls.bundle.js",
 	"report.bundle.js",
 	"telemetry.bundle.js",
+	"onboarding.bundle.js",
```

Comment:
This should be loaded async if tour are pending. 

Since this is only required once, it doesn't make much sense to load it always?